lib/order.rb

```diff
@@ -0,0 +1,71 @@
+require_relative 'customer.rb'
+
+class Order
+
+  attr_reader :id
+  attr_accessor :products, :customer, :fulfillment_status
```

Comment:
If you don't need to change these outside of the class, I'd just make them `_reader`s.